commit 0b3ce90e74f4d063432170239f6a3aea0c2ad665 (HEAD -> dev/sane)
Author: Colin <colin@uninsane.org>
Date:   2023-07-16 12:45:49 +0000

    sxmo_rotate: don't restart lisgd on sway

    AFAICT we restart lisgd because it learns the orientation via its
    `-o <orientation>` CLI argument. but we only pass that flag on dwm/x11,
    not sway. restarting it on sway is wasteful, and potentially racy (i'm
    able to get multiple lisgd's running just by enabling autorotate and
    flipping the phone rapidly), so don't do that.

diff --git a/scripts/core/sxmo_rotate.sh b/scripts/core/sxmo_rotate.sh
index 159236f..3a95657 100755
--- a/scripts/core/sxmo_rotate.sh
+++ b/scripts/core/sxmo_rotate.sh
@@ -52,7 +52,6 @@ xorgrotinvert() {
 
 swayrotinvert() {
 	swaymsg -- output "-" transform 180
-	superctl restart sxmo_hook_lisgd
 	sxmo_hook_rotate.sh invert
 	exit 0
 }
@@ -68,7 +67,6 @@ xorgrotnormal() {
 
 swayrotnormal() {
 	swaymsg -- output "-" transform 0
-	superctl restart sxmo_hook_lisgd
 	sxmo_hook_rotate.sh normal
 	exit 0
 }
@@ -84,7 +82,6 @@ xorgrotright() {
 
 swayrotright() {
 	swaymsg -- output "-" transform 90
-	superctl restart sxmo_hook_lisgd
 	sxmo_hook_rotate.sh right
 	exit 0
 }
@@ -100,7 +97,6 @@ xorgrotleft() {
 
 swayrotleft() {
 	swaymsg -- output "-" transform 270
-	superctl restart sxmo_hook_lisgd
 	sxmo_hook_rotate.sh left
 	exit 0
 }
